xml/System.IO/FileSystemInfo.xml

```diff
@@ -1283,8 +1283,6 @@ On Unix platforms that do not support creation or birth time, this property retu
 ## Remarks  
  `FileSystemInfo.Refresh` takes a snapshot of the file from the current file system. `Refresh` cannot correct the underlying file system even if the file system returns incorrect or outdated information. This can happen on platforms such as Windows 98.  
   
```

Comment:
I looked at the code again, and I think this line should be reverted.

Both FileInfo and DirectoryInfo get their data refreshed when retrieving attributes, if the data has been invalidated (-1).
We do invalidate our data when we make changes via code, but if the user did not make any invocations that may have changed state, the data could've still been updated externally, so the user should still call Refresh.

```suggestion
  Calls must be made to `Refresh` before attempting to get the attribute information, or the information will be outdated.  
  
```